src/util.ts

```diff
@@ -26,3 +24,9 @@ export function serializeAxiosError(
     data,
   };
 }
+
+export interface SerializedError {
```

Comment:
[Optional] A tiny remark  maybe we can call this `SerializedAxiosError` or `SerializedHttpError` as it has some very specific properties

Reply:
Yeah ...I'll do that